src/App.jsx

```diff
@@ -14,57 +15,56 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodName, setGoodName] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const reset = () => {
+    setGoodName('No goods');
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodName === 'No goods'
+          ? 'No goods selected'
+          : `${goodName} is selected`}
+        <button
+          data-cy="ClearButton"
+          type="button"
+          className="delete ml-3"
+          onClick={reset}
+        />
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={
+                good === goodName ? 'has-background-success-light' : ''
```

Comment:
it is better to use libraries like classnames or clsx for conditional classnames